data/json/monsters.json

```diff
@@ -2491,7 +2491,7 @@
     "death_function":"MELT",
     "special_attacks":[["LONGSWIPE", 8]],
     "description":"Intense radiation has spurred a unique form of necrosis and regeneration, it is impossible to tell if this creature was ever human.",
-    "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "VIS50", "NOGIB"]
+    "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "VIS50", "NOGIB", "BONES"]
```

Comment:
Yeah that's just there so if the player were to figure out some means of preventing it, they'd still have bones like they ought to.